packages/@lwc/template-compiler/src/__tests__/fixtures/parsing-errors/invalid-template-attribute/multiple-attributes/actual.html

```diff
@@ -1,6 +1,6 @@
 <template>
     <section>
-        <template if:true={showItems} for:each={items} for:item="item" onfoo={handleFoo}>
+        <template if:true={showItems} for:each={items} for:item="item" onfoo={handleFoo} class="slds-m-around_medium">
```

Comment:
@pmdartus should something like this with a combo of valid LWC directives and invalid attributes throw an error in 242?

I think it should but it might lead to more regressions.

Reply:
> should templates with a combo of valid LWC directives and invalid attributes throw an error in 242?

It depends what is the current behavior. What should aim for is to:
- preserve the existing behavior
- log a warning in all the places where `<template>` is misused so we can address this in future releases.